src/axom/sidre/core/View.cpp

```diff
@@ -662,13 +684,13 @@ IndexType View::getOffset() const
     {
       SLIC_ERROR_IF(
         offset % bytes_per_elem != 0,
-        "Unsupported operation on View "
-        << getPathName() << ".  Sidre assumes that offsets are"
-        << " given as integral number of elements into the array."
-        << " In this case, the offset was " << offset
-        << " bytes and each element is " << bytes_per_elem
-        << " bytes. If you have a need for non-integral offsets,"
-        << " please contact the Sidre team");
+        SIDRE_VIEW_LOG_PREPEND
+        << "Unsupported operation. "
```

Comment:
I understand and agree with the need for this check.  However, I think that an error in a const getter such as `getOffset()` or `getStride()` will be very surprising to a user.  This is a good error message, but for even greater clarity, please make two changes: 
- print `"Error calculating stride"` instead of `"Unsupported operation"`
- print `"In this View, the offset is "` instead of `"In this case, the offset was "`

Please make analogous changes to the `SLIC_ERROR_IF` message in `getStride()` as well.

Reply:
Thanks @agcapps -- I'll update the messages.